docs/acton-by-example/src/functions/actor_methods.md

```diff
@@ -31,6 +35,47 @@ actor main(env):
 Output:
 ```sh
 Computing result based on our secret 42
-Multiplying 3 with 42
+Calculator multiplying 3 with 42
 Result: 126
 ```
+
+## The implicit `self` in actors
+
+Unlike classes where methods must explicitly declare `self` as their first parameter, actors have an implicit `self` that is automatically available. This fundamental difference means that actor methods are defined without a `self` parameter. In fact, using `self` as a parameter name in actor methods or in the actor's initialization arguments will result in an error.
+
+The implicit `self` in an actor refers to the actor instance itself. This allows an actor to pass a reference to itself to another actor by simply passing `self`. This is particularly useful when implementing callback patterns or when an actor needs to register itself with another actor.
```

Comment:
I don't quite like this wording. It focuses on `self` is implicit, but `self` is just not needed to refer to attributes on the own actor.

The sole purpose of `self` in actor is to get an external view of the actor that we can pass to other actors that need to interact with us. I think we can express this much more concisely.